### DISCUSSION TITLE
04/070-%E7%AC%AC%E4%B8%83%E5%8D%81%E7%AB%A0-%E6%97%A0%E5%BD%A2%E4%BA%A4%E6%8D%A2

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#73] [General]
# 04/070-%E7%AC%AC%E4%B8%83%E5%8D%81%E7%AB%A0-%E6%97%A0%E5%BD%A2%E4%BA%A4%E6%8D%A2

在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。

https://tts.liuli.moe/04/070-%E7%AC%AC%E4%B8%83%E5%8D%81%E7%AB%A0-%E6%97%A0%E5%BD%A2%E4%BA%A4%E6%8D%A2.html

**Comment by xtw136** [comment]
这个翻译急需校对，人名都搞错了

**Reply by rxliuli** [reply]
什么人名？ @iolliotier

**Reply by iolliotier** [reply]
哪个？是思裔的名字还是Seo Si‐won西时元？这里我采取的翻译是徐诗妍，之前的西时元其实是错误翻译，Seo Si‐won是韩国的谚文罗马音，转换为谚文是서시원，在中文中서叫徐，시是诗，원可以翻译为妍或者媛，还有一个Won Min‐ji也是，总之读起来有种常凯申的既视感。之前考虑过Seo Si‐won继续采用旧译法，但还是改为这种了，如果不反对，我也可以把译名表和前文的人名都换为这个

**Reply by SamLukeYes** [reply]
@iolliotier 我记得麻美的战术电脑之前一直采用的是意译（机械娘）而不是音译（玛琪娜）

**Reply by rxliuli** [reply]
> @iolliotier 我记得麻美的战术电脑之前一直采用的是意译（机械娘）而不是音译（玛琪娜）

确实是这样。为什么突然修改了译名？ @iolliotier 另外已添加译名到译名对照表中，参考
https://github.com/liuli-moe/to-the-stars/commit/aa353f1a7da11d70d4b5213e2da373b165a2cf5a

**Reply by rxliuli** [reply]
> 哪个？是思裔的名字还是Seo Si‐won西时元？这里我采取的翻译是徐诗妍，之前的西时元其实是错误翻译，Seo Si‐won是韩国的谚文罗马音，转换为谚文是서시원，在中文中서叫徐，시是诗，원可以翻译为妍或者媛，还有一个Won Min‐ji也是，总之读起来有种常凯申的既视感。之前考虑过Seo Si‐won继续采用旧译法，但还是改为这种了，如果不反对，我也可以把译名表和前文的人名都换为这个

@iolliotier 优先保持前后文的译名一致性，修改译名请在单独的 PR 或 Issue 中解决，就像之前的 [**悲叹之种 => 悲叹立方**](https://github.com/liuli-moe/to-the-stars/pull/41)。吾辈将检查并暂时改回它们，否则前后文的不一致令人困惑。

**Reply by iolliotier** [reply]
我之前也没看出machina什么意思，之前以为这个是外国女性人名，没想到是机械娘？我在下次PR一起提交更改

**Comment by WHY-Nannmonee** [comment]
想问一下我在贴吧翻到了tts还有几篇同人作也在b站发布翻译了。但是贴吧给个b站链接都挂了。请问现在还能在哪看到吗？Clarisse Quest和艾玛的故事）感谢！！！！

**Reply by rxliuli** [reply]
b 站如果删除了可能就从互联网上消失了，这也是这个项目（以及在 liuli-moe 组织下其他几个项目）要解决的问题。如果你有死链接的话可以去 https://archive.org/ 看看，有可能被缓存过

**Reply by rxliuli** [reply]
吾辈在另一个网站看到了，你愿意帮忙将它们整理过来以便其他喜欢的人能更容易找到么？吾辈可以帮你初始化一个项目，并生成对应的网站和 epub

https://www.baihewenku.com/xiaoshuo/202305/1248388/

**Reply by WHY-Nannmonee** [reply]
好的，但是我具体需要怎么做：>

**Reply by rxliuli** [reply]
> 好的，但是我具体需要怎么做：>

项目已经初始化完成 <https://github.com/liuli-moe/clarisse-quest>，网站也已上线，你需要在 books/zh-CN 目录中补充内容，每个章节为一个 md 文件，例如 001.md 是第一章，002.md 是第二章，英文原作已经被拉取到 books/en-US，可以在校对发现问题时参考。
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/c852b236-548c-4eb4-8b18-e030c13607fc">

---

@CeciliaXXXZ  Ok，你已经接受邀请可以直接修改文件了。你可以现在 GitHub 网页上直接修改，修改完之后会自动更新网站，发布新的 epub。如果你对高级操作感兴趣，例如一次修改多个文件，请后续了解 Git 及 GitHub。

编辑按钮
<img width="646" alt="image" src="https://github.com/user-attachments/assets/beae39c2-e8dd-4641-be14-254674453460">
编辑页面
<img width="642" alt="image" src="https://github.com/user-attachments/assets/dcce4522-f3fd-47d8-a5df-5ab5f2fff537">
提交
<img width="649" alt="image" src="https://github.com/user-attachments/assets/76d5e7b6-2d5e-4476-b728-c24c3194ba84">

创建文件的按钮
<img width="648" alt="image" src="https://github.com/user-attachments/assets/12d8a314-0a97-4171-87c4-d8c2cb0f8f5a">
编辑，与上面编辑唯一不同的是需要填写文件名，后续编辑提交步骤一致
<img width="647" alt="image" src="https://github.com/user-attachments/assets/38577d3c-3980-43b4-9098-1e0e6ab5c9b1">

**Reply by rxliuli** [reply]
@CeciliaXXXZ 
网站地址：https://clarisse-quest.liuli.moe/001.html
Epub 下载：https://github.com/liuli-moe/clarisse-quest/releases/latest
